src/main/java/org/broadinstitute/hellbender/utils/gcs/BucketUtils.java

```diff
@@ -112,6 +113,29 @@ public static OutputStream createFile(String path, PipelineOptions popts) {
     }
 
     /**
+     * Open a binary file for writing regardless of whether it's on GCS, HDFS or local disk.
+     * For writing to GCS it'll use the application/octet-stream MIME type.
+     *
+     * @param path the GCS or local path to write to. If GCS, it must start with "gs://", or "hdfs://" for HDFS.
+     * @param auth authentication information.
+     * @return an OutputStream that writes to the specified file.
+     */
+    public static OutputStream createFile(String path, AuthHolder auth) {
+        PipelineOptions popts = auth.asPipelineOptionsOldStyle();
```

Comment:
Mention in the comments that this method will only work with `AuthHolders` created from an `apiKey` (if I interpret this correctly).